### ISSUE TITLE
[Bug]: Change the redis port: Error -3 connecting to redis:16379. Temporary failure in name resolution.

### ISSUE THREAD

**Issue by simonqian** [infiniflow/ragflow#4264]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### RAGFlow workspace code commit ID

fe279754ace6d44598d9051099b9ac0db58599fa

### RAGFlow image version

0.15.1

### Other environment information

```Markdown
OS: Ubuntu
deploy: docker
```

### Actual behavior

![image](https://github.com/user-attachments/assets/9b4a0a27-44cd-4aac-9ff4-b6efce51cb95)

### Steps to reproduce

```Markdown
I am using infiniflow/ragflow:v0.15.1-slim. Based on this version, I have modified the service_conf.yaml.template file and saved it as the infiniflow/ragflow:v0.15.2-slim mirror.

service_conf.yaml.template update:
...
redis:
  db: 1
  password: '${REDIS_PASSWORD:-infini_rag_flow}'
  host: '${REDIS_HOST:-redis}:${REDIS_PORT:-6379}'
...

.env file update:
...
DOC_ENGINE=${DOC_ENGINE:-infinity}
...
REDIS_PORT=16379
...
RAGFLOW_IMAGE=infiniflow/ragflow:v0.15.2-slim
...
```

### Additional information

![image](https://github.com/user-attachments/assets/32141230-079c-442e-9edd-4103f62abac8)

**Comment by KevinHuSh** [comment]
![image](https://github.com/user-attachments/assets/fd823c7c-eacf-45e8-b8f2-41aa9eb2a643)

**Comment by simonqian** [comment]
> ![image](https://private-user-images.githubusercontent.com/12318111/398877680-fd823c7c-eacf-45e8-b8f2-41aa9eb2a643.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyODQyMDQsIm5iZiI6MTczNTI4MzkwNCwicGF0aCI6Ii8xMjMxODExMS8zOTg4Nzc2ODAtZmQ4MjNjN2MtZWFjZi00NWU4LWI4ZjItNDFhYTllYjJhNjQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDA3MTgyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5OTE2MTRlNmU3Y2IzNDliZmQ1NzRlMzA1NWFlOTcyNzNkYzcwNmEwMGZiYmFjMjk3YmJiNTE2MWYzMzJjNWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.guFC7NpGO-dyS3XdmdVSOyaaUa81fvkxTfGQWGu9S30)

still same error
![image](https://github.com/user-attachments/assets/0a72e526-0382-472f-88c7-1bd2a2b3b575)

![image](https://github.com/user-attachments/assets/c4fa28f4-3b79-4b80-90ce-068aa481ff9b)

**Comment by KevinHuSh** [comment]
![image](https://github.com/user-attachments/assets/69acf054-2f58-4b28-9aa6-33b885bffeb7)

**Comment by simonqian** [comment]
> ![image](https://private-user-images.githubusercontent.com/12318111/398886774-69acf054-2f58-4b28-9aa6-33b885bffeb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyODY3MzUsIm5iZiI6MTczNTI4NjQzNSwicGF0aCI6Ii8xMjMxODExMS8zOTg4ODY3NzQtNjlhY2YwNTQtMmY1OC00YjI4LTlhYTYtMzNiODg1YmZmZWI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDA4MDAzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzMTY4YjAzZWMwMDYwMGUxMTFiNDFhNDI1YzQ0ZDhkMDRjMDQ3NTFkNmE4Mjc2N2IwYjI3OGY2NTM3OTZmZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8azGq7DazEFkwIh-PM0XgA7bGSly_HZiAZND9Bm_vyo)

Yes, I have modified the container name, but it does not affect this issue.

**Comment by KevinHuSh** [comment]
> > ![image](https://private-user-images.githubusercontent.com/12318111/398886774-69acf054-2f58-4b28-9aa6-33b885bffeb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyODY3MzUsIm5iZiI6MTczNTI4NjQzNSwicGF0aCI6Ii8xMjMxODExMS8zOTg4ODY3NzQtNjlhY2YwNTQtMmY1OC00YjI4LTlhYTYtMzNiODg1YmZmZWI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjI3VDA4MDAzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzMTY4YjAzZWMwMDYwMGUxMTFiNDFhNDI1YzQ0ZDhkMDRjMDQ3NTFkNmE4Mjc2N2IwYjI3OGY2NTM3OTZmZDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8azGq7DazEFkwIh-PM0XgA7bGSly_HZiAZND9Bm_vyo)
> 
> Yes, I have modified the container name, but it does not affect this issue.

I mean that's the reason. Revert it back.

**Comment by wangq8** [comment]
The bug has been long time, might not be valid in main branch, close it now. If you still met the issue, free free to reopen the bug.
